apps/server/src/modules/legacy-school/service/provisioning-options-update-service.ts

```diff
@@ -0,0 +1,42 @@
+import { Group, GroupService, GroupTypes } from '@modules/group';
+import { Injectable } from '@nestjs/common';
+import { EntityId } from '@shared/domain/types';
+import { AnyProvisioningOptions } from '../domain';
+
+@Injectable()
+export class ProvisioningOptionsUpdateService {
+	constructor(private readonly groupService: GroupService) {}
+
+	public async handleActions<T extends AnyProvisioningOptions>(
```

Comment:
I would prefer the naming handleUpdate and the specific provisioning options for schulconnex at this point. If we would have other options we shouldn't handle those options here